inc/settings.php

```diff
@@ -376,9 +376,147 @@ function vantage_siteorigin_settings_localize( $loc ){
 		// For the premium upgrade modal
 		'modal_title' => __('Vantage Premium Upgrade', 'vantage'),
 		'close' => __('Close', 'vantage'),
+
+		// For the settings metabox
+		'meta_box'            => __( 'Page settings', 'vantage' ),
+
+		// For archives section
+		'page_section_title' => __( 'Page Template Settings', 'vantage' ),
+		'page_section_description' => __( 'Change layouts for various pages on your site.', 'vantage' ),
+
+		// For all the different temples and template types
+		'template_home' => __( 'Blog Page', 'vantage' ),
+		'template_search' => __( 'Search Results', 'vantage' ),
+		'template_date' => __( 'Date Archives', 'vantage' ),
+		'template_404' => __( 'Not Found', 'vantage' ),
+		'template_author' => __( 'Author Archives', 'vantage' ),
+		'templates_post_type' => __( 'Type', 'vantage' ),
+		'templates_taxonomy' => __( 'Taxonomy', 'vantage' ),
 	);
 
 	return $loc;
 }
 endif;
-add_filter( 'siteorigin_settings_localization', 'vantage_siteorigin_settings_localize' );
\ No newline at end of file
+add_filter( 'siteorigin_settings_localization', 'vantage_siteorigin_settings_localize' );
+
+if( !function_exists('vantage_siteorigin_setting_update_image') ) :
+/**
+ * Convert post ID based images into full URLs
+ *
+ * @param $mods
+ *
+ * @return mixed
+ */
+function vantage_siteorigin_setting_update_image( $mods ) {
```

Comment:
We should actually be removing this with the latest version of the settings framework, but I'll make those changes when I update the submodule.